nautobot_chatops/utils.py

```diff
@@ -14,6 +14,61 @@
 logger = logging.getLogger(__name__)
 
 
+try:
+    from nautobot.core.celery import nautobot_task
+
+    @nautobot_task
+    def celery_worker_task(command, subcommand, params, dispatcher_module, dispatcher_name, context):
+        """Task executed by Celery worker.
+
+        Celery cannot serialize/deserialize objects, instead of passing the
+        function, registry or dispatcher_class, we will have to look them up.
+        """
+        # import done here instead of up top to prevent circular imports.
+        # Disabling cyclic-import as this does not affect the worker.
+        # pylint: disable=import-outside-toplevel,cyclic-import
+        from nautobot_chatops.workers import get_commands_registry
+
+        registry = get_commands_registry()
+        function = registry[command]["function"]
+        # Get dispatcher class from module
```

Comment:
Any further details here? I think I have an idea of what it is doing, but this may be something worth having additional comments on.